README.md

```diff
@@ -95,9 +97,9 @@ Make your project unique and memorable (most people only focus on technical aspe
     * Flow’s is yellow and gray
     * Make sure your docs and website follow this color theme
 - [x] Further reading
-  * If you want to learn more about marketing, see [LisaDziuba/Marketing-for-Engineers](https://github.com/LisaDziuba/Marketing-for-Engineers)
+  * If you want to learn more about marketing, see [LisaDziuba/Marketing-for-Engineers](https://github.com/LisaDziuba/Marketing-for-Engineers) and/or reach out to some open source marketing leaders such as [@kamcmahon](https://twitter.com/kamcmahon). [@jenlankford](https://twitter.com/jenlankford), [@amanda_katona](https://twitter.com/amanda_katona), [@FunnelFiasco](https://twitter.com/funnelfiasco) or [@Jen_L_Gray](https://twitter.com/jen_l_gray).
 
-## ✈️ Onboarding Methods
+## ✈️ Entry Barriers
```

Comment:
I think we should change the title to `Onboarding Users`. The overall section intent of the section is to convey that.